File: problems/0001-two-sum/python/solution.py
def twoSum(nums, target):
    """Two Sum — https://leetcode.com/problems/two-sum/"""
    seen = {}
    for i, n in enumerate(nums):
        j = seen.get(target - n)
        if j is not None:
            return [j, i]
        seen[n] = i
    return []

File: problems/0001-two-sum/typescript/solution.ts
// Two Sum — https://leetcode.com/problems/two-sum/
export function twoSum(nums: number[], target: number): number[] {
  const seen = new Map<number, number>();
  for (let i = 0; i < nums.length; i++) {
    const j = seen.get(target - nums[i]);
    if (j !== undefined) return [j, i];
    seen.set(nums[i], i);
  }
  return [];
}
